odk1-src/briefcase-using.rst

```diff
@@ -51,30 +51,81 @@ To download blank forms and completed form instances from an :doc:`Aggregate <ag
   
   .. container:: details
   
-    To speed up pulling forms from Aggregate,
-    enable :guilabel:`Pull submissions in parallel`
-    in the :guilabel:`Settings` tab.
+    To speed up pulling forms from Aggregate, enable :guilabel:`Pull submissions in parallel` in the :guilabel:`Settings` tab.
 
     .. image:: /img/briefcase-using/pull-in-parallel.*
 
-    However, if your Aggregate server is
-    :doc:`installed on Google App engine <aggregate-app-engine>`,
-    this setting may cause problems with large attachments.
-    If your form has submission attachments 
-    (file uploads, videos, images, sound recordings)
-    you should experiment with this setting
-    and see if it improves or worsens performance.
+    However, if your Aggregate server is :doc:`installed on Google App engine <aggregate-app-engine>`, this setting may cause problems with large attachments. If your form has submission attachments (file uploads, videos, images, sound recordings) you should experiment with this setting and see if it improves or worsens performance.
+
+.. _pull-from-collect:
+
+Pulling forms from ODK Collect
+------------------------------
+
+#. Ensure all filled-in forms are finalized.
+
+   If you have incomplete forms that you cannot finalize before pulling into Briefcase, delete them. If you need to keep them, make a copy of :file:`/sdcard/odk` before deleting them, and restore it after you are finished.
+
+#. Create a zip archive of the entire :file:`odk` directory.
+
+   .. tip::
+
+     You'll need to use an app for this.
+
+     One option is `OI File Manager <https://play.google.com/store/apps/details?id=org.openintents.filemanager>`_.
+
+#. Connect your Android device to your computer using a USB cable and choose to mount it as a Media device.
```

Comment:
> Connect your Android device to your computer using a USB cable. When prompted, choose to mount it as a Media device.

If the wording of the prompt is nearly universal across devices, it would be even better to say:

```rst
#. Connect your Android device to your computer using a USB cable. 
   When prompted, select :guilabel:`Mount as Media Device`.
```

or whatever the label text is.
But only if it is always the same.